docs/development/decision-records/2024-09-16-proposal-federated-catalog-distribution/README.md

```diff
@@ -0,0 +1,24 @@
+# Proposal for FederatedCatalog with Tractus-X distribution
+
+## Decision
+
+The proposed solution is to have a Federated Catalog as own runnable service capable of crawling all the chosen catalogs and expose that data.
+
+## Rationale
+
+The Federated Catalog aims at providing the aggregation of catalogs from multiple participants in a centralized point to reduce latency.
```

Comment:
this is not needed. No need to motivate the existence of the FC, only the decision to deploy it as standalone component.
It also makes sense here to highlight some of the disadvantages (increased complexity, more remote calls,...) and why they are still justified (scalability, etc.).

Reply:
Removed in [here](https://github.com/eclipse-tractusx/tractusx-edc/pull/1555/commits/48b1900fa819dbf0ec0f2652fc4fc07ef5cd8525).